grpc-api/src/main/java/io/deephaven/grpc_api/console/ScopeTicketResolver.java

```diff
@@ -64,10 +68,46 @@ public void forAllFlightInfo(final Consumer<Flight.FlightInfo> visitor) {
         return session.<T>nonExport().submit(() -> result);
     }
 
-    private Flight.FlightDescriptor descriptorForName(final String name) {
+    @Override
+    public <T> SessionState.ExportObject<T> resolve(final SessionState session,
+                                                    final Flight.FlightDescriptor descriptor) {
+        final String scopeName = nameForDescriptor(descriptor);
+        final ScriptSession gss = globalSessionProvider.getGlobalSession();
+        return session.<T>nonExport().submit(() -> gss.getVariable(scopeName));
+    }
+
+    @Override
+    public <T> SessionState.ExportBuilder<T> publish(final SessionState session,
+                                                     final ByteBuffer ticket) {
+        // We publish to the query scope after the client finishes publishing their result. We accomplish this by
+        // directly depending on the result of this export builder.
+        final SessionState.ExportBuilder<T> resultBuilder = session.nonExport();
+        final SessionState.ExportObject<T> resultExport = resultBuilder.getExport();
+        final SessionState.ExportBuilder<T> publishTask = session.nonExport();
+
+        final String varName = nameForTicket(ticket);
+        publishTask
+                .requiresSerialQueue()
+                .require(resultExport)
+                .submit(() -> {
+                    final ScriptSession gss = globalSessionProvider.getGlobalSession();
+                    gss.setVariable(varName, resultExport.get());
+                });
+
+        return resultBuilder;
+    }
+
+    public static Flight.Ticket ticketForName(final String name) {
+        final byte[] ticket = (TICKET_PREFIX + '/' + name).getBytes();
```

Comment:
`getBytes()` is using the platform default encoding. This is `UTF-8` on Linux, but it's latin 1 on most windows installs afaik. Pick UTF-8.